horton/gbasis/ints.h

```diff
@@ -74,11 +98,224 @@ class GB2NuclearAttractionIntegral: public GB2Integral {
         double* work_g2;
         double* work_boys;
     public:
-        GB2NuclearAttractionIntegral(long max_shell_type, double* charges, double* centers, long ncharge);
-        ~GB2NuclearAttractionIntegral();
-        virtual void add(double coeff, double alpha0, double alpha1, const double* scales0, const double* scales1);
+        /** @brief
+              Basic class for Nuclear attraction integrals.
```

Comment:
The documentation of the constructor should always be something of the following from: `Initialize a GB4Integral object.` or `Create a GB4Integral object.` The current wording would be good for the class documentation above.

Also further below, the`@param` lines for `work_g0` and the should can be removed.